Prevent timeouts during file sync & improve CI performance

larsewi opened the pull request:
Removed -c flag from rsync command which is likely causing timeouts by
computing checksums for all files. Default size/mtime comparison is
sufficient for the CI workspace synchronization and significantly
faster.

Signed-off-by: Lars Erik Wik <lars.erik.wik@northern.tech>

larsewi committed:
Removed rsync checksum flag to improve CI performance

Removed -c flag from rsync command which is likely causing timeouts by
computing checksums for all files. Default size/mtime comparison is
sufficient for the CI workspace synchronization and significantly
faster.

Signed-off-by: Lars Erik Wik <lars.erik.wik@northern.tech>

craigcomstock reviewed (approved):
Yes! Love it. I didn't even realize that this copy-back-to-host was here instead of in some mysterious jenkins bits. Good find!

larsewi commented:
AIX only:
[![Build Status](https://ci.cfengine.com/buildStatus/icon?job=pr-pipeline&build=12991)](https://ci.cfengine.com/job/pr-pipeline/12991/)

larsewi committed:
Added keep alive loop when syncing workspace

Syncing the workspace back after a job has ended takes a lot of time.
Hence, we need to print something every now and then to prevent Jenkins
from timing out.

Signed-off-by: Lars Erik Wik <lars.erik.wik@northern.tech>
